aiidalab_widgets_base/databases.py

```diff
@@ -475,7 +285,7 @@ def clean_up_database(self, database, plugin):
     def update(self, _=None):
         with self.hold_trait_notifications():
             database = requests.get(
-                "https://aiidateam.github.io/aiida-code-registry/database.json"
+                "https://aiidateam.github.io/aiida-code-registry/database_v2.json"
```

Comment:
I am not sure that the `hold_trait_notification()` context manager is really appropriate here. Are we changing multiple traits at once within the same thread? The hold should not be confused with a lock. See the [docs](https://traitlets.readthedocs.io/en/stable/using_traitlets.html#holding-trait-cross-validation-and-notifications) for reference.

Also, why not make the request outside of the hold?

_Since my comment is not directly related to this change set, please address it in a separate issue or PR if any action is deemed necessary._